.github/workflows/node-flow-pull-request-checks.yaml

```diff
@@ -203,7 +203,7 @@ jobs:
     needs:
       - dependency-check
       - spotless
-    if: ${{ github.event_name != 'pull_request' || (github.actor != 'dependabot[bot]' && github.event.pull_request.head.repo.full_name == github.repository) }}
+    if: ${{ github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
If we are re-enabling the dependency checks/JRS panel etc on dependabot PRs then i think this if clause changes to be the following:

`if: ${{ github.event_name != 'pull_request'`